doc/source/management/data_at_rest_encryption.rst

```diff
@@ -7,58 +7,207 @@ Data at Rest Encryption
+   Only one keyring plugin should be enabled at a time. Enabling multiple
+   keyring plugins is not supported and may result in data loss.
+
+.. seealso::
+
+   |MySQL| Documentation:
+      - `Installing a Keyring Plugin <https://dev.mysql.com/doc/refman/5.7/en/keyring-installation.html>`_
+      - `The --early-plugin-load Option <https://dev.mysql.com/doc/refman/5.7/en/server-options.html#option_mysqld_early-plugin-load>`_
+
+.. _data-at-rest-encryption.keyring.changing-default:
+
+Changing the Default Keyring Encryption
+================================================================================
+
+When encryption is enabled and the server is configured to use the KEYRING
+encryption, new tables use the default encryption key.
+
+You many change this default encryption via the
+:variable:`innodb_default_encryption_key_id` variable.
+
+.. seealso::
+
+   Configuring the way how tables are encrypted
+      :variable:`innodb_encrypt_tables`
+
+System Variables
+--------------------------------------------------------------------------------
+
+.. variable:: innodb_default_encryption_key_id
+
+   :version 5.7.23-24: Implemented
+   :cli: ``--innodb-default-encryption-key-id``
+   :dyn: Yes
+   :scope: Session
+   :vartype: Numeric
+   :default: 0
+
+The ID of the default encryption key. By default, this variable contains **0**
+to encrypt new tables with the latest version of the key ``percona_innodb-0``.
+
+To change the default value use the following syntax:
+
+.. code-block:: guess
+
+   mysql> SET innodb_default_encryption_key_id = NEW_ID
+
+Here, **NEW_ID** is an unsigned 32-bit integer.
+
+.. _data-at-rest-encryption.innodb-system-tablespace:
+
+InnoDB System Tablespace Encryption
+================================================================================
+
+The InnoDB system tablespace is encrypted by using master key encryption. The
+server must be started with the ``--bootstrap`` option.
+
+If the variable :variable:`innodb_sys_tablespace_encrypt` is set to ON and the
+server has been started in the bootstrap mode, you may create an encrypted table
+as follows:
+
+.. code-block:: guess
+
+   mysql> CREATE TABLE ... TABLESPACE=innodb_system ENCRYPTION='Y'
+
+.. note::
+
+   You cannot encrypt existing tables in the System tablespace.
+
+It is not possible to convert the system tablespace from encrypted to
+unencrypted or vice versa. A new instance should be created and user tables must
+be transferred to the desired instance.
+
+You can encrypt the already encrypted InnoDB system tablespace (key rotation)
+with a new master key by running the following ``ALTER INSTANCE`` statement:
+
+.. code-block:: guess
 
+   mysql> ALTER INSTANCE ROTATE INNODB MASTER KEY
+
+.. rubric:: Doublewrite Buffers
+
+The two types of doublewrite buffers used in |Percona Server| are encrypted
+differently.
+
+When the InnoDB system tablespace is encrypted, you can encrypt the ``doublewrite buffer``
```

Comment:
not "you can", but that it's just being encrypted as a part of the encrypted system tablespace

Reply:
Text updated as requested